db-commons/src/main/scala/org/bitcoins/db/DbManagement.scala

```diff
@@ -125,6 +126,16 @@ trait DbManagement extends BitcoinSLogger {
     flyway.info()
   }
 
+  def migrationsApplied(): Int = {
+    val applied = flyway.info().applied()
+    driver match {
+      case SQLite =>
+        applied.size
+      case PostgreSQL =>
+        applied.size - 1
```

Comment:
nit: Maybe add the comment i have in `DbManagementTest` so that we don't forget why we have a `-1` in the future?